### DISCUSSION TITLE
Leaderboard displays scores from before maps were ranked/qualified

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#19178] [Q&A] [answered]
Version 2022.716.1
Since the leaderboards were just updated, I'm opening a discussion rather than an issue.
https://osu.ppy.sh/b/3575465 was ranked 7 days ago; however, the leaderboards show a score 22 days old.

![osu_2022-07-16_22-45-23](https://user-images.githubusercontent.com/74262407/179385664-af302c74-d16e-4ee5-afdb-50da5582c83c.png)
![osu_2022-07-16_22-45-29](https://user-images.githubusercontent.com/74262407/179385667-169be598-df01-48bc-b7ee-ad219dffc89a.png)
![osu_2022-07-16_22-45-47](https://user-images.githubusercontent.com/74262407/179385668-3a1037ff-6cbd-41dd-9b94-1955127ae3bb.png)
Since lazer's demographic doesn't really target newly ranked maps, these are few and far between to find, but some other examples are https://osu.ppy.sh/b/3629655 https://osu.ppy.sh/b/3638104 and https://osu.ppy.sh/b/3649646

**Comment by peppy** [comment]
This is due to one missing piece of the puzzle, which is a process to handle beatmap approved status changes. It's on the infrastructure TODO list, and we will clean out any scores which are on an incorrect timeline when that work is completed.

**Comment by cihe13375** [comment]
This bug still exists e.g. https://lazer.ppy.sh/beatmapsets/2040593#mania/4256821 and probably should be resolved before lazer leaderboard goes live

**Reply by peppy** [reply, marked as answer]
This should already be resolved. The reason they are sticking around is that we're migrating systems and the deletion process is only running on the new storage.
